text/metrics/0117-min-max-last-sum-count-aggregator.md

```diff
@@ -0,0 +1,192 @@
+# Replace MinMaxSumCount aggregator with MinMaxLastSumCount aggregator
+
+Add a LastValue component to the MinMaxSumCount aggregator, name it MinMaxLastSumCount.
+
+## Motivation
+
+The MinMaxSumCount aggregator is specified as the default for grouping
+instruments (i.e., ValueRecorder and ValueObserver).  This default
+makes it impossible with these instruments to expose a Gauge
+containing the last value in Prometheus and Statsd, yet a last-value
+Gauge is the natural low-cost exposition format for these instruments.
+
+This proposal adds the LastValue to the Min, Max, Sum, and Count
+fields and renames the combined aggregator MinMaxLastSumCount.  Using
+this default, we are able to export the default aggregation over OTLP
+and expose the last value for Prometheus and Statsd exporters.
+
+## Explanation
+
+The Min, Max, Sum, and Count were selected as the default aggregation
+for GROUPING instruments because they are an inexpensive, mergeable
+summary of the value distribution and can be conveyed exactly using a
+fixed amount of space.  Any quantile value other than the Min and Max
+does not meet these requirements, either it will be inexact or require
+non-constant space.
+
+LastValue was not included because it does significantly add
+information about the distribution of values.  LastValue could be used
+to establish a tendency in the data (e.g., if the last value is
+greater than the average value, the signal is rising), but it could
+just be an outlier.  Knowing the _value_ part a LastValue may not tell
+us much about the distribution of values, but knowing the _timestamp_
+of the last value (i.e., the maximum timestamp) adds to our
+understanding of the timeseries.
+
+Adding the LastValue to MinMaxSumCount is partly pragmatic.  This
+makes it possible to use the default aggregator for grouping
+instruments, send data through OTLP, and expose the LastValue as a
+Gauge without loss of the information commonly used by other exporters
+(i.e., Min, Max, Sum, Count).
+
+## Internal details
+
+This proposal covers how the data should be represented in OTLP.
+
+Prior to this proposal, the MinMaxSumCount value has been exposed as a
+`Summary` value, which is a general-purpose value type meant for
```

Comment:
Are `Summary` values not able to be merged because of the percentiles?

Reply:
Yes. This is the reason why Summary values are generally discouraged in Prometheus--they are not mergeable while Histograms are.